lardata/ArtDataHelper/Dumpers/dump_pids.fcl

```diff
@@ -0,0 +1,79 @@
+#
+# File:     DumpParticleIDs.fcl
+# Purpose:  Dump on screen particle ID content
+# Author:   H. Greenlee
+# Date:     Oct. 14, 2021
+# Version:  1.0
+#
+# Service dependencies:
+# - message facility
+#
+
+process_name: DumpParticleIDs
+
+services: {
+  
+  message: {
+  #   debugModules: [ "*" ]
```

Comment:
This has been dropped since forever... please remove it (even if it's just a comment right now).
```suggestion
```

Reply:
Removed.